.kokoro/test-samples-impl.sh

```diff
@@ -33,7 +33,6 @@ export PYTHONUNBUFFERED=1
 env | grep KOKORO
 
 # Install nox
-# `virtualenv==20.26.6` is added for Python 3.7 compatibility
```

Comment:
should we remove `virtualenv==20.26.6` because 3.7 is removed too?

Reply:
We want virtualenv.
I am testing locally to see if we can simply install virtualenv without specifying a version number.

Reply:
Somewhere Owlbot.py did some post processing and reverted that bit of code to add that change back in because it pulled in the template for certain files and overwrote my changes.
I edited owlbot.py so that it will not touch certain templates AND so that for certain templates that it will protect the edits that we need to have go through.